redash/query_runner/cass.py

```diff
@@ -0,0 +1,137 @@
+import json
+import sys
+import logging
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+logger = logging.getLogger(__name__)
+
+class Cassandra(BaseQueryRunner):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            'type': 'object',
+            'properties': {
+                'host': {
+                    'type': 'string',
+                },
+                'port': {
+                    'type': 'number',
+                    'default': 9042,
+                },
+                'Keyspace': {
+                    'type': 'string',
+                    'title': 'Keyspace name'
+                },
+                'username': {
+                    'type': 'string',
+                    'title': 'Username'
+                },
+                'password': {
+                    'type': 'string',
+                    'title': 'Password'
+                }
+            },
+            'required': ['Keyspace']
+        }
+
+    @classmethod
+    def type(cls):
+        return "Cassandra"
+
+    @classmethod
+    def enabled(cls):
+        try:
+            from cassandra.cluster import Cluster
+        except ImportError:
+            return False
```

Comment:
You can move this whole block to the top of the file (to avoid duplicate import code), and set a variable (`enabled`) which you use here.  You can see the [BigQuery module](https://github.com/getredash/redash/blob/master/redash/query_runner/big_query.py#L17-L26) as an example.